data/secscan_model/secscan_v4_model.py

```diff
@@ -211,10 +212,25 @@ def needs_reindexing_query(indexer_hash):
             ),
         )
 
+        def manifest_unsupported_status(manifest):
```

Comment:
Perhaps rename to `mark_manifest_unsupported` to better represent the effect calling this function has.